tests/Optimizer2/Inspection/Inspection.cpp

```diff
@@ -77,9 +84,41 @@ TEST(Inspection, statust_test_deserialize_fail) {
     "fehler": 2
   })"_vpack;
 
-  auto res = deserializeWithStatus<Dummy>(testSlice);
+  auto res = deserializeWithErrorT<Dummy>(testSlice);
 
   ASSERT_FALSE(res.ok()) << fmt::format("Did not detect the error we exepct");
 
-  EXPECT_EQ(res.error(), "Found unexpected attribute 'fehler'");
+  EXPECT_EQ(res.error().error(), "Found unexpected attribute 'fehler'");
 }
+
+TEST(Inspection, arangodb_query_plan_to_inspectables) {
+  // TODO: This has been the original suited place for our inspectable plan
+  // tests but does not work right now "out of the box" as a lot of components
+  // are required to actually run them (e.g. QueryMock)
+  /*std::unique_ptr<arangodb::tests::graph::GraphTestSetup> s{nullptr};
+  std::unique_ptr<arangodb::tests::graph::MockGraphDatabase>
+  singleServer{nullptr}; std::shared_ptr<arangodb::aql::Query> query{nullptr};
+
+
+  singleServer =
+      std::make_unique<arangodb::tests::graph::MockGraphDatabase>(s->server,
+  "testVocbase");
+
+
+  query = singleServer->getQuery("RETURN 1", {"v", "e"});*/
+
+  /*
+  std::string queryString = "RETURN 1";
+  using MockQuery = arangodb::tests::mocks::MockQuery;
+  arangodb::tests::mocks::MockServer server{"PRMR_0001", true};
+  auto ctx = std::make_shared<arangodb::transaction::StandaloneContext>(
+      server.getSystemDatabase());
+  auto fakeQuery = std::make_shared<MockQuery>(ctx, queryString);
+*/
+}
```

Comment:
this needs cleanup - currently not used as I've had to write the tests I wanted to in "SingleServerProviderTest.cpp"